src/routeList.js

```diff
@@ -0,0 +1,29 @@
+// import React from 'react';
+import Order from './views/Order'
+import CustomerMap from './views/Map/CustomerMap'
+// import AgentMAp from './views/Map/AgentMAp'
+import Summary from './views/Order/Summary'
+import Address from './views/Order/Address'
+import Community from './views/Community'
+import History from './views/History'
+import Account from './views/Pages/Accout'
+import Edit from './views/Community/Editor'
+import View from './views/Community/View'
+import Page404 from './views/Pages/Page404'
+
+// https://github.com/ReactTraining/react-router/tree/master/packages/react-router-config
+const routes = [
+  { path: '/community/edit/:id', exact: true, name: 'Edit', component: Edit },
+  { path: '/order/summary', exact: true, name: 'Summary', component: Summary },
+  { path: '/order/address', exact: true, name: 'Address', component: Address },
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected a line break after this opening brace.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5703258)